src/components/OSCALSystemImplementation.test.js

```diff
@@ -27,35 +27,41 @@ export default function testOSCALSystemImplementation(
 
   test(`${parentElementName} shows component title`, () => {
     renderer();
-    const result = screen.getByText("Example Component");
+    const result = screen.getByLabelText("An example component.");
     expect(result).toBeVisible();
   });
 
   test(`${parentElementName} shows component description`, async () => {
     renderer();
-    await userEvent.hover(screen.getByText("Example Component"));
+    await userEvent.hover(screen.getByLabelText("An example component."));
     expect(
       await screen.findByText("An example component.")
     ).toBeInTheDocument();
   });
 
   test(`${parentElementName} shows component status`, () => {
     renderer();
-    const component = screen.getByText("Example Component").closest("tr");
+    const component = screen
+      .getByLabelText("An example component.")
+      .closest("tr");
     const result = within(component).getByText("operational");
     expect(result).toBeVisible();
   });
 
   test(`${parentElementName} shows component type`, () => {
     renderer();
-    const component = screen.getByText("Example Component").closest("tr");
+    const component = screen
+      .getByLabelText("An example component.")
+      .closest("tr");
     const result = within(component).getByText("software");
     expect(result).toBeVisible();
   });
 
   test(`${parentElementName} shows component version`, () => {
     renderer();
-    const component = screen.getByText("Example Component").closest("tr");
+    const component = screen
+      .getByLabelText("An example component.")
+      .closest("tr");
```

Comment:
With the addition of component titles appearing in Inventory Items, this text (the component title) cannot be found as easy with it displaying twice. I was thinking label would be the next best option.